view/bookactions.rb

```diff
@@ -0,0 +1,24 @@
+require_relative './actions'
+require_relative '../model/book'
+
+class BookActions
+  include Actions
+
+  def initialize(books)
+    @books = books
+  end
+
+  def read_all
+    @books.each_with_index { |book, index| puts "ID: #{index}, Book title: #{book.title}, Author: #{book.author}" }
+    wait
+  end
+
+  def create
+    puts 'Enter the info of the book you want to add:'
+    print 'Title:'
+    title = gets.chomp
+    print 'Author:'
+    author = gets.chomp
+    @books << Book.new(title, author)
+  end
```

Comment:
- [ ] great job on cleaning up the main.rb file 👍 however the `create` method is not in line with the SOLID principle in particular the single responsibility rule. please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) provided by Microverse for more information.